.circleci/config.yml

```
@@ -96,6 +96,8 @@ jobs:
     <<: *defaults
     steps:
       - checkout
+      - run: git submodule sync
+      - run: git submodule update --init
```

```suggestion
      - checkout_with_submodules
```
How about bundling the repeatable steps into a command that can be re-used, wdyt?